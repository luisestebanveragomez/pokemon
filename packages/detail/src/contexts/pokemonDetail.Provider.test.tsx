import { render } from '@testing-library/react';
import { PokemonDetailProvider, usePokemonDetailContext } from './pokemonDetail.Provider';

const TestingComponent = () => {
  const data = usePokemonDetailContext();
  return (
    <>
      <p></p>
    </>
  );
};

describe("SectionPokemon Detail Provider", () => {
  it("Should Render", () => {
    const { container } = render(
      <PokemonDetailProvider>
        <TestingComponent />
      </PokemonDetailProvider>
    );
    expect(container).toMatchSnapshot()
  });
});
