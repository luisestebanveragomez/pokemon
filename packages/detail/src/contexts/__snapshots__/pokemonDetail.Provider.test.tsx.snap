// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionPokemon Detail Provider Should Render 1`] = `
<div>
  <p />
</div>
`;
